Tasks/KubernetesManifestV0/src/actions/bake.ts

```diff
@@ -12,12 +13,25 @@ import { Helm, NameValuePair } from 'kubernetes-common-v2/helm-object-model';
 import * as TaskParameters from '../models/TaskInputParameters';
 import { KomposeInstaller } from '../utils/installers';
 import * as utils from '../utils/utilities';
+import * as DeploymentHelper from '../utils/DeploymentHelper';
+import * as TaskInputParameters from '../models/TaskInputParameters';
 
 abstract class RenderEngine {
     public bake: () => Promise<any>;
     protected getTemplatePath = () => {
         return path.join(getTempDirectory(), 'baked-template-' + uuidV4() + '.yaml');
     }
+    protected updateImages(filePath: string) {
+        if (TaskInputParameters.containers) {
+            const updatedFilesPaths: string[] = DeploymentHelper.updateResourceObjects([filePath], [], TaskInputParameters.containers);
+            let fileContents: string[] = [];
+            updatedFilesPaths.forEach((path) => {
+                const content = yaml.safeDump(JSON.parse(fs.readFileSync(path).toString()));
+                fileContents.push(content);
+            });
+            fs.writeFileSync(filePath, fileContents.join("\n---\n"));
```

Comment:
Is there no function in the yaml lib to do the concatenation for you?

Reply:
Couldn't find anything. I tried doing yaml.safeDump on array of JSON objects. But that results in a yaml file without any separator `---`.

Reply:
Add a test case where you use read the generated file. i.e. if `yaml.safeLoad` works